controller/config.go

```diff
@@ -354,7 +352,11 @@ func (c Config) AuditLogMaxSizeMB() int {
 // files to keep.
 func (c Config) AuditLogMaxBackups() int {
 	if value, ok := c[AuditLogMaxBackups]; ok {
-		return value.(int)
+		if intValue, ok := value.(int); ok {
+			return intValue
+		}
+		return int(value.(float64))
```

Comment:
This seems like it's in the wrong layer. This should be done in the networking layer.

Reply:
The other getters that return ints do the same thing, although most of them use `mustInt` - I don't use it because that for this because it panics on 0, which is an acceptable value here.